include/fastcdr/config.h.in

```diff
@@ -46,4 +46,12 @@
 #define __BIG_ENDIAN__ @__BIG_ENDIAN__@
 #endif
 
+#if defined(__ARM_ARCH) && __ARM_ARCH <= 7
+#define FASTCDR_32BIT
+#endif
+
+#if defined(_WIN32)
```

Comment:
I think this isn't enough, `_WIN32` is defined in Win64 too.

```suggestion
#if defined(_WIN32) and not defined(_WIN64)
```

I think the suggestion ensures we are in a pure Win32 platform.

Reply:
I just committed this change, but in this PR I switched over all cases of `if defined(_WIN32)`. Those seemed intentional, and now would not be hit by WIN64 builds.

The build with this stricter check makes the Windows build unstable - it looks like some of the `#if defined(_WIN32)` blocks were not specific to 32bit builds.

Reply:
What do you mean? `and` and `not` are *alternative operators* and should be supported by all C++11 compliant compilators.
I personally prefer using them only in the preprocessor, but it's a matter of style. In any case, shouldn't be incorrect. Please, correct me if I'm wrong.

Reply:
Hmm... maybe there was a non-ascii character in there then. The Windows builds complained specifically about that line. My mistake

Reply:
I just tested this explicitly with a commit. GCC accepts the alternative operators, but the Windows build preprocessor does not.

Reply:
We did some testing here too. Seems that Visual Studio needs to set a flag in the compiler options to allow them.
It is OK to use traditional operators to simplify the overall process.
Thank you.